### PR TITLE
fix torch_quant ci

### PR DESCRIPTION
**Pull request by chenbohua3**

### CODE REVIEW

File: tools/torch_quant/torch_quant/observer.py [hunk 1]
```diff
@@ -214,24 +251,30 @@ def forward(self, x):
 
 
 class BiasObserver(Observer):
-    def __init__(self, w_ob: Observer, act_ob: Observer,
-                 dtype: torch.dtype = torch.qint32,
-                 qscheme: torch.qscheme = torch.per_tensor_affine, **kwargs) -> None:
-        super().__init__(**kwargs)
+    def __init__(self, w_ob: Observer, act_ob: Observer, **kwargs) -> None:
+        dtype = torch.qint32
+        is_w_per_channel = is_per_channel(w_ob.qscheme)
```
Comment (deeply) [inline tools/torch_quant/torch_quant/observer.py:256]: just use `w_ob.per_channel`?

File: tools/torch_quant/torch_quant/observer.py [hunk 2]
```diff
@@ -66,10 +98,16 @@ def __init__(self, dtype, qscheme, **kwargs) -> None:
         self.register_buffer('eps', torch.tensor([torch.finfo(torch.float32).eps]))
         self.observe = True
         self.fake_quant = True
+        self.ch_axis = ch_axis
+        self._register_load_state_dict_pre_hook(partial(pre_load_state_dict_hook, self))
 
     @property
     def symmetric(self) -> bool:
-        return self.qscheme in (torch.per_tensor_symmetric, torch.per_channel_symmetric)
+        return is_per_channel(self.qscheme)
```
Comment (deeply) [inline tools/torch_quant/torch_quant/observer.py]: symmetric doesn't always equal to per_channel?

Comment (deeply) [reply tools/torch_quant/torch_quant/observer.py]: never mind, I see the new commit fix this.

File: tools/torch_quant/torch_quant/observer.py (hunk 1 above)
Comment (chenbohua3) [reply tools/torch_quant/torch_quant/observer.py:259]: Generally, only the qscheme of weight will be per-channel.
Have added activation qscheme to co-determine the qscheme of bias

**Comment by chenbohua3** [comment]
A todo has been added to remind us to find a more efficient way to deal with the shape of the scale

File: tools/torch_quant/torch_quant/observer.py (hunk 1 above)
Comment (deeply) [reply tools/torch_quant/torch_quant/observer.py:259]: ok, but code can be simplified as `if w_ob.per_channel or act_ob.per_channel:` ?

Comment (chenbohua3) [reply tools/torch_quant/torch_quant/observer.py]: done

**Review by deeply — APPROVED** [review]
LGTM

**Pull request merged**